sklearn/manifold/t_sne.py

```diff
@@ -545,15 +545,19 @@ class TSNE(BaseEstimator):
         least 200.
 
     n_iter_without_progress : int, optional (default: 30)
+        Only used if method='exact'
         Maximum number of iterations without progress before we abort the
-        optimization.
+        optimization. If method='barnes_hut' this parameter is fixed to
```

Comment:
This behavior is somewhat peculiar and we could work around it, but let's leave it for now.